### DISCUSSION TITLE
Parse date exception for MySQL

### DISCUSSION THREAD

**Discussion by etekhno** [jOOQ/jOOQ#15180] [Q&A] [answered]
Schema.getTables() for MySQL throws cannot parse default expression: 2020-12-30 [*]00:00:00. 

![image](https://github.com/jOOQ/jOOQ/assets/78665413/17ccfa1f-3621-49e9-ba29-b4dc1fb676dc)

**Comment by lukaseder** [comment, marked as answer]
Thanks for your message. How can this be reproduced? I tried this with the latest jOOQ version:

```sql
create table t (d datetime default '2020-12-30 00:00:00');
```

And then:

```java
System.out.println(ctx.meta().getTables("t"));
```

It prints:

```
["test"."t"]
```

**Reply by etekhno** [reply]
Please try with the MariaDB dialect.

**Reply by etekhno** [reply]
create table temp (yourday datetime default '1899-12-30 00:00:00');

List<Table<?>> tables = sourceSchema.get().getTables();

As I have multiple Schema in the database. I'm using JOOQ 3.16.6 version.

![image](https://github.com/jOOQ/jOOQ/assets/78665413/7cd0a893-5edf-4426-9dd7-5a12d3776662)

**Reply by lukaseder** [reply]
Just tried it with the latest version of jOOQ, without any problems.

I'll be happy to look at a fully self contained reproducer. We have a template here to create such reproducers: https://github.com/jOOQ/jOOQ-mcve

Please do try the latest version as well, first, though. Even for the 3.16 minor release, 3.16.6 is far from the latest patch version.
